server/src/services/auth.service.ts

```diff
@@ -103,6 +105,52 @@ export class AuthService extends BaseService {
     return mapUserAdmin(updatedUser);
   }
 
+  async createPincode(auth: AuthDto, { pincode }: createPincodeDto): Promise<UserAdminResponseDto> {
+    const user = await this.userRepository.getByEmail(auth.user.email, false, true);
```

Comment:
With another argument for `getByEmail` now I think passing an `options` object would be nicer. Like this it's just `false, true` and nobody knows what it is, whereas `{ withPincode: true }` would make it explicit :)

Reply:
Try that and it create many irrelevant changes in this PR. i plan do it separately

Reply:
Awesome! Separate PR and then rebasing on that sounds perfect.